input/docs/controls/ColorPicker.md

```diff
@@ -0,0 +1,492 @@
+string nameOfMyColor = MahApps.Metro.Controls.ColorHelper.GetColorName(myColor, theDictionaryToUse);
+```
+
+## Getting a color from a given name
+You can also get back the color by looking up its name. The routine will first check the dictionary for the first occurrence of the given name and if it was not found it will try to use [`ColorConverter.ConvertFromString`](docs.microsoft.com/en-us/dotnet/api/system.windows.media.colorconverter.convertfromstring) which also accepts the HTML-Notation of the `Color`. If the `Color` cannot be found it will return null. Blow you can find some examples.
+
+```
+// Note: If you set the second parameter to 'null' the default dictionary will be used.  
+Color? myColor = MahApps.Metro.Controls.ColorHelper.ColorFromString(myColorName, theDictionaryToUse);
+
+// this will look up the German word "Blau" and returns blue color
+Color? myColor = MahApps.Metro.Controls.ColorHelper.ColorFromString("Blau", null);
+
+// this will look up the HTML-notation "#FF000000" and returns black color
+Color? myColor = MahApps.Metro.Controls.ColorHelper.ColorFromString("#FF000000", null);
+```
+
+
+## How to provide custom color names
+You can change the build in dictionary via `MahApps.Metro.Controls.ColorHelper.ColorNamesDictionary` by adding, removing or changing any dictionary entry. You can also create your own `Dictionary<Color?,string>` anywhere in your `Model` or `ViewModel` to provide your own color names. That way you can use also different dictionaries for different `ColorPicker`. 
+
+Example:
+```c#
+Dictionary<Color?,string> myColorNames = new Dictionary<Color?,string>()
+{
+    { Colors.Green, "every thing is fine" }, 
+    { Colors.Yellow, "warning" },
+    { Colors.Red, "error" }
+};
+```
+## Provide translations for a language of your choice
+You can help providing translations to the build in dictionary. Currently implemented language:
+- English
+- German
+
+To do so we recommend using the [ResXManager](https://marketplace.visualstudio.com/items?itemName=TomEnglert.ResXManager) and please read [this guide](https://github.com/MahApps/MahApps.Metro/wiki/Contributing)
+
+# ColorCanvas
+The `ColorCanvas` lets the user select a `Color` by either manipulating the ARGB-Values, manipulating the HSV-Values, entering the color name or HTML-notation or picking a color from the screen
+
+## The user interface
+![](images/ColorPicker_ColorCanvas_Numbered.png)
+
+| No. | Description                              |
+|-----|------------------------------------------|
+| 01  | red color channel from 0 to 255          |
+| 02  | green color channel from 0 to 255        |
+| 03  | blue color channel from 0 to 255         |
+| 04  | hue channel from 0° to 360°              |
+| 05  | saturation channel from 0% to 100%       |
+| 06  | value channel from 0% to 100 %           |
+| 07  | alpha channel from 0 to 255              |
+| 08  | the color name or html-notation          |
```

Comment:
Maybe case sensitive?
red color -> Red color
green color -> Green color
...

Reply:
You are the boss. I can do this.